### DISCUSSION TITLE
License for pre-trained models also Apache?

### DISCUSSION THREAD

**Discussion by dyollb** [Project-MONAI/model-zoo#139] [Q&A]
I was looking at the pre-trained model https://github.com/Project-MONAI/model-zoo/tree/dev/models/wholeBrainSeg_Large_UNEST_segmentation to see what the license condition is for the pre-trained model. Is this model (and all others) licensed as part of the MONAI model zoo package or do separate conditions apply? I guess it may be complicated, since the training data comes from different sources...

Formulated as a feature request: *"as a user I would like to know the license conditions of each model in the zoo"*.

**Comment by yiheng-wang-nv** [comment]
HI @dyollb , thanks for posting this question. So far, we updated the license related descriptions and changed the format of each bundle within the repository. Could you please check the latest doc:
https://github.com/Project-MONAI/model-zoo/tree/0367326a85882302331fef757228d84e0abe984c#license
